### DISCUSSION TITLE
Meter advances from top instead of bottom

### DISCUSSION THREAD

**Discussion by jbelew** [jomjol/AI-on-the-edge-device#2601] [Q&A]
I've been trying to figure out why I've been getting a number of bad reads with negative rate issues and finally came across this page -- https://jomjol.github.io/AI-on-the-edge-device-docs/Watermeter-specific-analog---digital-transition/. From the image there it looks like the system is designed for meters that advance from the bottom instead of the top, like my old PG&E electricity meter. Is the some expert setting that I'm missing, or should I file a bug / feature request?

**Comment by caco3** [comment]
Can you explain a bit more, what you mean with "bottom instead of the top"?

The physical orientation does not have any impact. Eg. I also read my gas meter with it and the camera points to the wall instead to the floor.
